### DISCUSSION TITLE
Missing agent.js in 1.34.0

### DISCUSSION THREAD

**Discussion by olivergardiner** [community/community#127183] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I've just signed up for Copilot and wanted to use it with QT. I installed NeoVIM and Node and cloned the Copilot plugin per the instructions, which pulled down the current master branch at 1.34.0. When I tried to configure the Copilot plugin for QT it couldn't find the path to agent.js. On further investigation and some Googling, the reason became clear - it is no longer present in the latest repo. I added agent.js and agent.js.map from the 1.33.0 zip and all now works fine. Has agent.js been deprecated?

**Comment by knightno1** [comment]
Thank you so much, I also encountered this problem, finally solved, thank you. Big lao.

**Reply by AyinVee** [reply]
Same here need more guidelines

**Reply by olivergardiner** [reply]
Hi AyinVee,

Until there is advice from the authors, I'd recommend going to the list of tags here: https://github.com/github/copilot.vim/tags , downloading the zip for 1.33.0 and simply unzipping that into $HOME/AppData/Local/nvim/pack/github/start/copilot.vim instead of using Git to clone the repo

**Reply by knightno1** [reply]
# you are so cool

**Reply by tpope** [reply, marked as answer]
`agent.js` was renamed to `language-server.js`.
